config_example.sh

```diff
@@ -654,19 +654,33 @@ set -x
 
 # Set a single config variable AGENT_E2E_TEST_SCENARIO to create a cluster for the different scenarios 
 # i.e. Single Node Openshift(SNO), Highly Available (HA) or Compact cluster.
+# The boot mode for the agent machines can only be set to ISO or PXE.
+# For backward compatibility of CI jobs, the default boot mode is ISO.
 # The only supported values for AGENT_E2E_TEST_SCENARIO are 
-# - COMPACT_IPV4
-# - COMPACT_IPV6
-# - COMPACT_IPV4_DHCP
-# - COMPACT_IPV6_DHCP
-# - HA_IPV4
-# - HA_IPV6
-# - HA_IPV4_DHCP
-# - HA_IPV6 _DHCP
-# - SNO_IPV4
-# - SNO_IPV6
-# - SNO_IPV4_DHCP
-# - SNO_IPV6_DHCP
+# - COMPACT_IPV4_ISO
```

Comment:
I wonder if boot mode should be included in AGENT_E2E_TEST_SCENARIO. The more options we include in AGENT_E2E_TEST_SCENARIO the longer this list becomes. What do people think about having BOOT_MODE separate from AGENT_E2E_TEST_SCENARIO?

Reply:
That's a good suggestion!

Reply:
I think if we are going to implement this, then the env var name should be `AGENT_E2E_TEST_BOOT_MODE`

Reply:
That sounds fine to me. I think that, in general, we could distinguish two broader set of configuration variables, primary and secondary ones.
Primary config vars are core traits, and they could be an aggregate var (like for example the topology), and they definited inline. Secondary variables capture maybe some less frequently used config traits, and could be defined separately.